esbuild.cjs

```diff
@@ -2,8 +2,9 @@
 const { build } = require('esbuild');
 build({
     bundle: true,
-    minify: true,
+    minify: false, // Cannot minify due to https://github.com/node-fetch/node-fetch/issues/784
```

Comment:
I think the [keep names](https://esbuild.github.io/api/#keep-names) option should fix this issue, without disabling minification

Reply:
Haha, I did try it yesterday but it didn't seem to work, VSCode find only found 3 instances of AbortSignal in the code with and without minify enabled, So I thought it wasn't working..

But I just tried with grep and it shows 7 vs 9 with keepNames.
Guess VSCode tricked me.

Reply:
This bug was a real elusive one as the error message doesn't say much and the cache saving was working properly.